tests/code_fellows/test_linked_list.py

```diff
@@ -1,17 +1,20 @@
 # https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
 
 import pytest
-
+import sys
 from code_fellows.linked_list import LinkedList
+from io import StringIO
```

Comment:
Nice find on the test for print!
these imports should be arranged per pep008 https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

> You should put a blank line between each group of imports.

so in this case:
```python
import sys
from io import StringIO

import pytest

from code_fellows.linked_list import LinkedList
```

Reply:
might see if VS code has the plugin `isort` https://pypi.org/project/isort/